docs/pages/access-controls/guides/headless-login.mdx

```diff
@@ -0,0 +1,237 @@
+
+```code 
+$ tsh ls --headless --proxy=example --user=user
+# Complete headless authentication in your local web browser:
+# 
+# https://proxy.example.com:3080/web/headless/86172f78-af7c-5935-a7c1-ed06b94f17dc
+# 
+# or execute this command in your local terminal:
+# 
+# tsh headless approve --user=dev --proxy=proxy.example.com:3080 86172f78-af7c-5935-a7c1-ed06b94f17dc
+# # User approves through link
+# Node Name Address        Labels                                                                             
+# --------- -------------- -----------
+# server01  127.0.0.1:3022 arch=x86_64
+#
+$ tsh ssh --headless --proxy=example --user=user server01
+# Complete headless authentication in your local web browser:
+# 
+# https://proxy.example.com:3080/web/headless/864cccd9-2425-46d9-a9f2-636387e66ebf
+# 
+# or execute this command in your local terminal:
+# 
+# tsh headless approve --user=dev --proxy=proxy.example.com:3080 864cccd9-2425-46d9-a9f2-636387e66ebf
+# # User approves through link
+```
+
+## Additional Options
+
+### Set headless login as default
+
+To make headless login the default authentication method for your Teleport Cluster, add `connector_name: headless` to your cluster configuration:
+
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+<Tabs>
+  <TabItem label="Static Config">
+    Auth Server `teleport.yaml` file:
+
+    ```yaml
+    auth_service:
+      authentication:
+        type: local
+        second_factor: on
+        webauthn:
+          rp_id: example.com
+        connector_name: headless # headless by default
+    ```
+  </TabItem>
+  <TabItem label="Dynamic resources">
+    Create a `cap.yaml` file or get the existing configuration using
+    `tctl get cluster_auth_preference`:
+
+    ```yaml
+    kind: cluster_auth_preference
+    version: v2
+    metadata:
+      name: cluster-auth-preference
+    spec:
+      type: local
+      second_factor: "on"
+      webauthn:
+        rp_id: example.com
+      connector_name: headless # headless by default
+    ```
+
+    Update the configuration:
+
+    ```code
+    $ tctl create -f cap.yaml
+    # cluster auth preference has been updated
+    ```
+  </TabItem>
+</Tabs>
+</ScopedBlock>
+
+<ScopedBlock scope={["cloud"]}>
+Create a `cap.yaml` file or get the existing configuration using
+`tctl get cluster_auth_preference`:
+
+```yaml
+kind: cluster_auth_preference
+version: v2
+metadata:
+  name: cluster-auth-preference
+spec:
+  type: local
+  second_factor: "on"
+  webauthn:
+    rp_id: example.com
+  connector_name: headless # headless by default
+```
+
+Update the configuration:
+
+```code
+$ tctl create -f cap.yaml
+# cluster auth preference has been updated
+```
+</ScopedBlock>
+
+### Set environment variable
+
+Users can also set the following environment variables to reduce the number of flags passed to `tsh`:
+
+- `TELEPORT_PROXY`
+- `TELEPORT_USER`
+- `TELEPORT_HEADLESS`
```

Comment:
Can you explain what each variable does and give some examples? Should I set them to 1, "on", true when I want to use them?